### PR TITLE
Feature issue 288 unique customdomains on mobilization

### PR DESCRIPTION
**Pull request by lpirola**

### CODE REVIEW

**Comment by luizalbsilva** [comment]
Issue #288

**Comment by igr-santos** [comment]
@luizalbsilva and @lpirola  slug should be unique also, today I receive an error with status 422 but without message

File: spec/requests/mobilizations_spec.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+RSpec.describe "Mobilizations", type: :request do
+  let!(:community) { create :community }
+  let!(:logged_user) { create :user, admin: true }
+
+  before { stub_current_user(logged_user) }
+
+  describe "POST /mobilizations" do
+
+    context "repeated custom_domain" do
+      let(:mobilization) { create :mobilization, custom_domain: 'egaliteliberteetfraternite.fr' }
```
Comment (lpirola) [inline spec/requests/mobilizations_spec.rb]: @luizalbsilva great custom domain! :)

File: spec/models/mobilization_spec.rb [hunk 2]
```diff
@@ -44,10 +54,10 @@
 
   context "create mobilization from TemplateMobilition object" do
```
Comment (lpirola) [inline spec/models/mobilization_spec.rb]: @luizalbsilva maybe add a text to show error message when slug is not unique?

Comment (igr-santos) [inline spec/models/mobilization_spec.rb]: Yes!

**Pull request merged**